docs/incremental-xmlupload.md

```diff
@@ -85,3 +86,28 @@ dsp-tools xmlupload --incremental additional_data_replaced_[timestamp].xml
 | <center>Important</center>                                                                                                                      |
 |-------------------------------------------------------------------------------------------------------------------------------------------------|
 | Internal IDs and IRIs cannot be mixed within the same file. An XML file uploaded with the incremental option must not contain any internal IDs. |
+
+
+
+## 4. Continue an interruped xmlupload
+
+If an xmlupload didn't finish successfully, 
+some resources have already been created, but others not.
+If the remaining resources have references to created ones,
+these references must be made with IRIs,
+and the created resources must be removed from the XML file
+(otherwise they would be created a second time).
+
```

Comment:
If the resources not uploaded to the DSP-API and reference those, which were successfully created, the original reference IDs must be replaced with the new IRIs.